prime-router/metadata/hl7_mapping/STLTs/ignore/ignore.yml

```diff
@@ -0,0 +1,14 @@
+hl7Type: ORU_R01
+hl7Version: 2.5.1
+extends: ../../ORU_R01/ORU_R01-base
```

Comment:
Is this really necessary?  This is not a schema we would want in production.

Reply:
Done, didn't realize this was one of the prod-facing files!